### DISCUSSION TITLE
How to link to another file

### DISCUSSION THREAD

**Discussion by lukepighetti** [artempyanykh/marksman#139] [Q&A] [answered]
I have two Markdown files in the same directory

`NOTES.md`

```md
# Notes

This is a bunch of notes
```

`PEOPLE.md`

```md
# People

This is a bunch of people that are referenced in [[NOTES.md]]
```

I have come to realize that you cannot link by filenames by default. That's fine, but I cannot figure out how we're supposed to link to other files at all. None of the following seem to work for me

- `[[NOTES.md]]`
- `[[NOTES]]`
- `[[notes]]`
- `[[NOTES.md#Notes]]`
- `[Notes](NOTES.md)`

Can someone provide a concrete example of how to link to other documents?

**Comment by artempyanykh** [comment, marked as answer]
@lukepighetti please check https://github.com/artempyanykh/marksman#workspace-folders-project-roots-and-single-file-mode It might be that you don't have a workspace folder set up. The easiest way is to drop an empty `.marksman.toml` file at the root folder of your notes, e.g. `touch ~/Documents/.marksman.toml`

Regarding completion by file that the Config docs: https://github.com/artempyanykh/marksman/blob/main/docs/configuration.md There's a config option to make wiki-link completion suggest filenames instead of titles.
